### ISSUE TITLE
about what accuracy should be record 

### ISSUE THREAD

**Issue by YuGuilliman** [Wang-ML-Lab/TSDA#4]
Hellow!
I am now doing my ERM experiment. I would like to ask what an ideal curve should look like after the experiment (whether stable or both are acceptable). As for the results presented in the paper, should the top one be from a stable curve? Or if it's acceptable to choose the top one from an unstable curve, which one should I record as the result in the paper (point 1 or 2 or something else)? There should be two figures below if everything is okay.
Best wishes for your kind help!

![6bef026359caaede3845d5b6878b792](https://github.com/Wang-ML-Lab/TSDA/assets/79433082/5b04d16c-1697-434c-a0cc-81f8ab2385cc)
![35e04b1257db1ccd7e4f0ef66cc4eb9](https://github.com/Wang-ML-Lab/TSDA/assets/79433082/225f5872-f85b-4ac7-a1b5-2304bbf4aed4)

**Comment by shsjxzh** [comment]
Hi YuGuilliman,

An ideal curve should look like figure 1. In our paper, we use the last epoch's accuracy as the model's performance. To stabilize your training, you could consider reducing the lambda_gan parameter, which will decrease the impact of adversarial learning on the overall model.

Best,
Zihao

**Comment by YuGuilliman** [comment]
Thank you for your help!

**Comment by YuGuilliman** [comment]
Thank you for your kind help again!
I would like to ask another question that does your tgt_acc curve always get higher when both the epoch_num and the src_acc increase, which means the tgt_acc curve does not fall down like the curve in figure 1(stable curve)?

**Comment by shsjxzh** [comment]
No, it not always get higher, but good hyperparameters can make it more stable. For convenience and fairness, we always report the final epoch's accuarcy.

Best,
Zihao

**Comment by YuGuilliman** [comment]
I think i  do understand, thank you！
